packages/fxa-auth-server/lib/routes/utils/oauth.js

```diff
@@ -57,9 +57,11 @@ module.exports = {
     credentials.tokenVerified = true;
     credentials.client = await oauthdb.getClientInfo(clientId);
 
-    // The following upsert gets no `deviceInfo`.
     // However, `credentials.client` lets it generate a default name for the device.
-    await devices.upsert(request, credentials, {});
+    await devices.upsert(request, credentials, {
```

Comment:
Nit: The "However" in the comment above is now confusing, I think we should move it down with the other comment and say something like:

// Connect the new refreshToken to the existing device record, or create a new placeholder if there isn't one.
// We don't provide any other device info, but `credentials.client` will let it generate a default name for the device.

Reply:
👍